experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -4,67 +4,52 @@
 
 import 'package:flutter/material.dart';
 
-class ComponentScreen extends StatelessWidget {
-  const ComponentScreen({super.key, required this.showNavBottomBar});
+class FirstComponentList extends StatelessWidget {
+  const FirstComponentList({
+    super.key,
+    required this.showNavBottomBar,
+    required this.scaffoldKey,
+    required this.showSecondList,
+  });
 
   final bool showNavBottomBar;
+  final GlobalKey<ScaffoldState> scaffoldKey;
+  final bool showSecondList;
 
   @override
   Widget build(BuildContext context) {
-    return Expanded(
-      child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 10),
-        child: Align(
-          alignment: Alignment.topCenter,
-          child: SizedBox(
-            width: maxWidthConstraint,
-            child: ListView(
-              shrinkWrap: true,
-              children: [
-                colDivider,
-                colDivider,
-                const Buttons(),
-                colDivider,
-                colDivider,
-                const IconToggleButtons(),
-                colDivider,
-                const FloatingActionButtons(),
-                colDivider,
-                const Chips(),
-                colDivider,
-                const Cards(),
-                colDivider,
-                const TextFields(),
-                colDivider,
-                const Dialogs(),
-                colDivider,
-                const Switches(),
-                colDivider,
-                const Checkboxes(),
-                colDivider,
-                const Radios(),
-                colDivider,
-                const ProgressIndicators(),
-                colDivider,
-                showNavBottomBar
-                    ? const NavigationBars(
-                        selectedIndex: 0,
-                        isExampleBar: true,
-                      )
-                    : Container(),
-              ],
-            ),
-          ),
-        ),
-      ),
+    return ListView(
+      shrinkWrap: true,
+      children: [
+        const Actions(),
+        const Communication(),
+        const Containment(),
+        Navigation(scaffoldKey: scaffoldKey),
+        if (!showSecondList) ...[const Selection(), const TextInputs()],
+      ],
+    );
+  }
+}
+
+class SecondComponentList extends StatelessWidget {
+  const SecondComponentList({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView(
+      shrinkWrap: true,
```

Comment:
It looks like it is being used here too. @QuncCccccc are these used inside of other lists, or are they being placed in Columns? These should probably be SliverLists instead